### ISSUE TITLE
`make build` fails: pytest-forked-1.4.0 not supported for interpreter python2.7

### ISSUE THREAD

**Issue by dnmfarrell** [urbit/urbit#5833]
**Describe the bug**
Running the `make` command fails. Here is the output with `--show-trace`:

```
nix-build -A urbit -A herb --no-out-link --show-trace
error: while evaluating the attribute 'buildInputs' of the derivation 'herb' at /nix/store/yj9bkfxsvv5n9mhgwq3bsi3r2amjdr8a-nixpkgs-src/pkgs/stdenv/generic/make-derivation.nix:205:7:
while evaluating the attribute 'out.outPath' at /nix/store/yj9bkfxsvv5n9mhgwq3bsi3r2amjdr8a-nixpkgs-src/lib/customisation.nix:204:13:
while evaluating the attribute 'propagatedBuildInputs' of the derivation 'python2.7-requests-2.27.1' at /nix/store/yj9bkfxsvv5n9mhgwq3bsi3r2amjdr8a-nixpkgs-src/pkgs/stdenv/generic/make-derivation.nix:205:7:
while evaluating the attribute 'out.outPath' at /nix/store/yj9bkfxsvv5n9mhgwq3bsi3r2amjdr8a-nixpkgs-src/lib/customisation.nix:204:13:
while evaluating the attribute 'propagatedBuildInputs' of the derivation 'python2.7-urllib3-1.26.2' at /nix/store/yj9bkfxsvv5n9mhgwq3bsi3r2amjdr8a-nixpkgs-src/pkgs/stdenv/generic/make-derivation.nix:205:7:
while evaluating the attribute 'dev.outPath' at /nix/store/yj9bkfxsvv5n9mhgwq3bsi3r2amjdr8a-nixpkgs-src/lib/customisation.nix:204:13:
while evaluating the attribute 'nativeBuildInputs' of the derivation 'python2.7-cryptography-3.3.2' at /nix/store/yj9bkfxsvv5n9mhgwq3bsi3r2amjdr8a-nixpkgs-src/pkgs/stdenv/generic/make-derivation.nix:205:7:
while evaluating the attribute 'out.outPath' at /nix/store/yj9bkfxsvv5n9mhgwq3bsi3r2amjdr8a-nixpkgs-src/lib/customisation.nix:204:13:
while evaluating the attribute 'nativeBuildInputs' of the derivation 'python2.7-hypothesis-4.57.1' at /nix/store/yj9bkfxsvv5n9mhgwq3bsi3r2amjdr8a-nixpkgs-src/pkgs/stdenv/generic/make-derivation.nix:205:7:
while evaluating the attribute 'out.outPath' at /nix/store/yj9bkfxsvv5n9mhgwq3bsi3r2amjdr8a-nixpkgs-src/lib/customisation.nix:204:13:
while evaluating the attribute 'propagatedBuildInputs' of the derivation 'python2.7-pytest-xdist-1.34.0' at /nix/store/yj9bkfxsvv5n9mhgwq3bsi3r2amjdr8a-nixpkgs-src/pkgs/stdenv/generic/make-derivation.nix:205:7:
while evaluating 'getOutput' at /nix/store/yj9bkfxsvv5n9mhgwq3bsi3r2amjdr8a-nixpkgs-src/lib/attrsets.nix:598:23, called from undefined position:
while evaluating anonymous function at /nix/store/yj9bkfxsvv5n9mhgwq3bsi3r2amjdr8a-nixpkgs-src/pkgs/stdenv/generic/make-derivation.nix:170:13, called from undefined position:
while evaluating 'callPackageWith' at /nix/store/yj9bkfxsvv5n9mhgwq3bsi3r2amjdr8a-nixpkgs-src/lib/customisation.nix:117:35, called from /nix/store/yj9bkfxsvv5n9mhgwq3bsi3r2amjdr8a-nixpkgs-src/pkgs/top-level/python-packages.nix:7925:19:
while evaluating 'makeOverridable' at /nix/store/yj9bkfxsvv5n9mhgwq3bsi3r2amjdr8a-nixpkgs-src/lib/customisation.nix:67:24, called from /nix/store/yj9bkfxsvv5n9mhgwq3bsi3r2amjdr8a-nixpkgs-src/lib/customisation.nix:168:34:
while evaluating anonymous function at /nix/store/yj9bkfxsvv5n9mhgwq3bsi3r2amjdr8a-nixpkgs-src/pkgs/development/python-modules/pytest-forked/default.nix:1:1, called from /nix/store/yj9bkfxsvv5n9mhgwq3bsi3r2amjdr8a-nixpkgs-src/lib/customisation.nix:69:16:
while evaluating 'makeOverridablePythonPackage' at /nix/store/yj9bkfxsvv5n9mhgwq3bsi3r2amjdr8a-nixpkgs-src/pkgs/top-level/python-packages.nix:29:37, called from /nix/store/yj9bkfxsvv5n9mhgwq3bsi3r2amjdr8a-nixpkgs-src/pkgs/development/python-modules/pytest-forked/default.nix:11:1:
while evaluating 'makeOverridable' at /nix/store/yj9bkfxsvv5n9mhgwq3bsi3r2amjdr8a-nixpkgs-src/lib/customisation.nix:67:24, called from /nix/store/yj9bkfxsvv5n9mhgwq3bsi3r2amjdr8a-nixpkgs-src/pkgs/top-level/python-packages.nix:31:12:
while evaluating anonymous function at /nix/store/yj9bkfxsvv5n9mhgwq3bsi3r2amjdr8a-nixpkgs-src/pkgs/development/interpreters/python/mk-python-derivation.nix:30:1, called from /nix/store/yj9bkfxsvv5n9mhgwq3bsi3r2amjdr8a-nixpkgs-src/lib/customisation.nix:69:16:
pytest-forked-1.4.0 not supported for interpreter python2.7
make: *** [Makefile:4: build] Error 1
```

**To Reproduce**
Steps to reproduce the behaviour:
1. Checkout master or cdd2b7902677e7d66fdd5d24569f313a2b38fb05 (HEAD).
2. Run `make`

**Expected behaviour**
The build should succeed.

**System (please supply the following information, if relevant):**
 - OS: Ubuntu 20.04

**Additional context**
Make build succeeds on urbit-v1.8. Git bisect identifies 442c18ee5b244a42e99d5fc13de48db165275465 as the first breaking commit.

**Notify maintainers**
@joemfb

**Comment by dnmfarrell** [comment]
Ah deleting `-A herb` fixes it, as in https://github.com/urbit/urbit/issues/5816.

**Comment by dnmfarrell** [comment]
Fixed in 0018d2c33b37e69f830ad66dbc77bdb566cb2ef5.

**Comment by ohAitch** [comment]
Does herb run on python 2.7?
